app/views/checkout/thankyou.scala.html

```diff
@@ -13,11 +13,18 @@
 
 @main("Confirmation | Digital | Subscriptions | The Guardian", bodyClasses=List("is-wide"), touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
 
-<main class="page-container gs-container gs-container--slim"
-data-tracking-prop17="GuardianDigiPack:Order Complete"
-data-tracking-products="Subscriptions and Membership;GUARDIAN_DIGIPACK;@{subscriptionProduct.frequency.numberOfMonths};@{subscriptionProduct.price};purchase">
->
-
+<script type="text/javascript">
+    guardian.slug="GuardianDigiPack:Order Complete";
```

Comment:
Yeah maybe, I think it's worth the slight abuse of MV* in exchange for the clarity of a single place where we expose global properties. Plus it encourages scripts in the middle of the page, which can cause blocking if we're not careful.